creator/ingest_runs/models.py

```diff
@@ -1,13 +1,17 @@
 import hashlib
 import uuid
+
 from django.db import models
 from django.contrib.auth import get_user_model
+import django_rq
+from rq.job import NoSuchJobError, JobStatus
 
 from creator.files.models import Version
 from creator.jobs.models import JobLog
 
 DELIMITER = "-"
 NAME_PREFIX = "INGEST_REQUEST"
+INGEST_QUEUE_NAME = 'ingest'
```

Comment:
@Gio can you move this into the settings files and then import it in this module